.github/workflows/android-build-test-fabric.yml

```diff
@@ -0,0 +1,38 @@
+name: Test Android build on Fabric
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'android/**'
+      - 'common/**'
+      - 'src/fabric/**'
+      - 'FabricExample/**'
+      - 'package.json'
+      - 'RNScreens.podspec'
```

Comment:
Could you tell why is this line needed?

Reply:
You are right
There is no need for podspec on Android
It is just copy paste mistake

Reply:
Fixed in [2bbc895](https://github.com/software-mansion/react-native-screens/pull/1321/commits/2bbc8959824ed82c836501608ef66df7140e7e7e)